docs/sovereign/other-vm.md

```diff
@@ -0,0 +1,6 @@
+# Other-VM
+
+:::note
+As the MultiversX Sovereign Chains ecosystem grows, additional VMs will be added and described here over time. Each new VM will follow the same integration pattern: assigning a unique base address, implementing the appropriate interfaces (such as `VMExecutionHandler`), and supporting cross-VM calls via the `blockchainHook` mechanism.
```

Comment:
actually SpaceVM implements every necessary interface and the new VM needs to only change the EXECUTOR part inside the SpaceVM.

Reply:
:+1: